charts/latest/csi-driver-smb/values.yaml

```diff
@@ -43,3 +43,6 @@ windows:
       repository: mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar
       tag: v1.2.1-alpha.1-windows-1809-amd64
       pullPolicy: IfNotPresent
+
+kubelet:
+  path: /var/lib/kubelet
```

Comment:
actually there is windows kubelet too, by default it's `C:\var\lib\kubelet`, so it could better be:
```
kubelet:
  linux_path: /var/lib/kubelet
  windows_path: 'C:\var\lib\kubelet'
```

Reply:
Ok. What value should I specify for `volumes.hostPath`? Earlier we have only one(`Kubelet.path`) but now we will have to values: `kubelet.linuxPath` and `Kubelet.windowsPath`. Is there's a way to conditionally take values in helm based on Node Types(windows or linux)?

Or Else can have one more parameter called: `node.os` (supported values: `windows`, `linux`). If the `node.os` is `windows`, then we use windows paths or else we use linux paths.

Reply:
you only need to change hostPath field here:
https://github.com/kubernetes-csi/csi-driver-smb/blob/99b7bbe09cacc91bf2fa5ee2614c2ed11a392cf3/charts/latest/csi-driver-smb/templates/csi-smb-node-windows.yaml#L129-L140